service/src/main/java/greencity/service/EventServiceImpl.java

```diff
@@ -122,23 +124,25 @@ public void delete(Long eventId, String email) {
     public EventDto getEvent(Long eventId, Principal principal) {
         Event event =
             eventRepo.findById(eventId).orElseThrow(() -> new NotFoundException(ErrorMessage.EVENT_NOT_FOUND));
-        EventDto eventDto = modelMapper.map(event, EventDto.class);
+        EventDto eventDto;
```

Comment:
we do not need it. We could use `return` [here](https://github.com/ita-social-projects/GreenCity/pull/6267/files#diff-eee18a7834701a9c0911786df843dbb6c25712682bc01c3c89e8d2b61f16e158R130) or [here](https://github.com/ita-social-projects/GreenCity/pull/6267/files#diff-eee18a7834701a9c0911786df843dbb6c25712682bc01c3c89e8d2b61f16e158R132)